crates/phactory/src/lib.rs

```diff
@@ -249,6 +250,10 @@ pub struct Phactory<Platform> {
 
     #[serde(skip)]
     trusted_sk: bool,
+
+    #[serde(skip)]
+    #[serde(default = "Instant::now")]
+    uptime: Instant,
```

Comment:
Maybe call it `started_at`? Since `uptime` sounds more like a duration.

Reply:
Done